pandas_gbq/gbq.py

```diff
@@ -767,6 +770,116 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
     return final_df
 
+def from_gbq(query, project_id=None, index_col=None, col_order=None,
```

Comment:
this should simply replace ``read_gbq``, changing the top-level API is a non-starter.